impls/quiver-arrow-qpid-proton-c.c

```diff
@@ -319,6 +321,14 @@ static bool handle(struct arrow* a, pn_event_t* e) {
         fail_if_condition(e, pn_listener_condition(pn_event_listener(e)));
         break;
 
+    case PN_PROACTOR_TIMEOUT:
+      FAIL("timeout expired %lf", a->desired_duration);
+      /* stop(a); */
+      // NOTE: stop(a) works correctly but causes the arrow to exit
+      // with 0 status, which makes quiver think it succeeded and gives
```

Comment:
Exit code 0 is what I was going for.  The idea is to run with --duration 2m and get normal program termination after said time.